app/templates/road-sign-concepts/new.hbs

```diff
@@ -7,51 +7,75 @@
     </LinkTo>
   </BreadcrumbsItem>
 
-  <NavigationBar />
+  <NavigationBar/>
```

Comment:
What formatter are you using? It seems to do some weird thing imo :smile:

Reply:
yeah I was thinking about the same, probably my IDE is messing up. I will commit from the command line next so it should fix it